Kinvey-Xamarin/Client/Client.cs

```diff
@@ -194,7 +194,7 @@ public Builder(string appKey, string appSecret)
 				Credential currentCredential = this.Store.GetStoredCredential(this.ssoGroupKey);
 				if (currentCredential != null)
 				{
-					await User.LoginAsync(currentCredential, c);
+					User.LoginAsync(currentCredential, c);
```

Comment:
@vinaygahlawat I thought you were planning to separate this into a "sync" login / setup user method. Is `User.loginAsync(credential)` effectively the same thing?